src/System.IO.FileSystem.DriveInfo/tests/System.IO.FileSystem.DriveInfo.Tests.csproj

```diff
@@ -21,7 +20,7 @@
       https://github.com/dotnet/corefx/issues/564
       https://github.com/dotnet/corefx/issues/592
     -->
-    <RunTestsForProject>False</RunTestsForProject>
+    <RunTestsForProject>True</RunTestsForProject>
```

Comment:
You can remove this. Please also add: <TestCategories>InnerLoop;OuterLoop</TestCategories>